test/test-creation.js

```diff
@@ -6,749 +6,759 @@ var assert = require('yeoman-assert');
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/security/social/CustomSignInAdapter.java',
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/service/SocialService.java',
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/web/rest/SocialController.java',
+    SERVER_TEST_SRC_DIR + 'com/mycompany/myapp/repository/CustomSocialUsersConnectionRepositoryIntTest.java',
+    SERVER_TEST_SRC_DIR + 'com/mycompany/myapp/service/SocialServiceIntTest.java'
+  ],
+
+  jwt: [
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/security/jwt/JWTConfigurer.java',
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/security/jwt/JWTFilter.java',
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/security/jwt/TokenProvider.java',
+  ],
+
+  gateway: [
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/config/GatewayConfiguration.java',
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/gateway/ratelimiting/RateLimitingFilter.java',
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/gateway/ratelimiting/RateLimitingRepository.java',
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/web/rest/dto/RouteDTO.java',
+    SERVER_MAIN_SRC_DIR + 'com/mycompany/myapp/web/rest/GatewayResource.java',
+    CLIENT_MAIN_SRC_DIR + 'app/admin/gateway/gateway.controller.js',
+    CLIENT_MAIN_SRC_DIR + 'app/admin/gateway/gateway.state.js',
+    CLIENT_MAIN_SRC_DIR + 'app/admin/gateway/gateway.html',
+    CLIENT_MAIN_SRC_DIR + 'app/admin/gateway/gateway.routes.service.js',
+    DOCKER_DIR + 'registry.yml'
+  ]
 };
 
 describe('JHipster generator', function () {
-    this.timeout(5000); //to avoid occassional timeouts
-
-    describe('default configuration', function () {
-        beforeEach(function (done) {
-            helpers.run(path.join(__dirname, '../generators/app'))
-                .withOptions({skipInstall: true})
-                .withPrompts({
-                    "baseName": "jhipster",
-                    "packageName": "com.mycompany.myapp",
-                    "packageFolder": "com/mycompany/myapp",
-                    "authenticationType": "session",
-                    "hibernateCache": "ehcache",
-                    "clusteredHttpSession": "no",
-                    "websocket": "no",
-                    "databaseType": "sql",
-                    "devDatabaseType": "h2Memory",
-                    "prodDatabaseType": "mysql",
-                    "useSass": false,
-                    "enableTranslation": true,
-                    "buildTool": "maven",
-                    "rememberMeKey": "5c37379956bd1242f5636c8cb322c2966ad81277",
-                    "searchEngine": "no"
-                })
-                .on('end', done);
-        });
-
-        it('creates expected default files', function () {
-            assert.file(expectedFiles.server);
-            assert.file(expectedFiles.maven);
-            assert.file(expectedFiles.client);
-            assert.file(['gulpfile.js']);
-        });
+  this.timeout(5000); //to avoid occassional timeouts
```

Comment:
Hm, sorry, looks like my IDE applied my own style here... Will fix this when I get to work!